src/core/types/addons.js

```diff
@@ -173,6 +173,8 @@ export type AddonType = {|
   isRestartRequired: boolean,
   isWebExtension: boolean,
   themeData?: ThemeData,
+  // This is present for collection add-ons
+  collectionNotes?: string | null,
```

Comment:
I don't think we should be doing this, because it weakens our `AddonType` IMO, meaning the Flow checker will be less accurate and potentially miss issues.

Since this is only used in the "collections" context, that would probably make sense to have a new type:

```js
type CollectionAddonType = {|
  ...AddonType,
  notes: string | null,
|};
```

What do you think?

Reply:
I also think it would make sense to do it here.

Reply:
I originally tried to create a new type for `CollectionAddonType`, but it was causing Flow nightmares. I agree it would be better, so I'll try again. 👍